content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -136,6 +137,31 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
+### Danger Zone Setting Configuration
```

Comment:
BTW, also suggest adding the paragraphs in https://longhorn.io/docs/1.6.0/references/settings/#danger-zone. Thank you.

Reply:
Added. Thanks.

Reply:
The table and some paragraphs are missing.

Reply:
Added the tables and other paragraphs. Thanks